mud-contracts/world/src/codegen/world/ISmartStorageUnit.sol

```diff
@@ -16,7 +17,7 @@ interface ISmartStorageUnit {
 
   function eveworld__createAndAnchorSmartStorageUnit(
     uint256 smartObjectId,
-    EntityRecordData memory entityRecordData,
+    EntityRecordTableData memory entityRecordData,
```

Comment:
I dont think this should be changed. external Interface requires `EntityRecordData`

Reply:
that was an artifact, sorry
changing this now